metrics/bigfiles_high_water_mark

```diff
@@ -1 +1 @@
-1094
+1143
```

Comment:
See note below about the growth in date_input_style.py and what we can do in the future now that we have better tests...

```
389: tests/integration/*****
381: records_mover/db/postgres/copy_options/date_input_style.py
373: setup.py

Reduce total number of bigfiles violations to 1094 or below!

Tasks: TOP => quality
(See full trace by running task with --trace)
Existing violations: 1094
Found 1143 bigfiles violations
```